assets/javascripts/modules/listCollapse.js

```diff
@@ -0,0 +1,61 @@
+/**
+  ListCollapse
+
+  Expand/collapse list content on smaller breakpoints, remain inline non interactive list at larger breakpoints.
+
+  Mandatory JS hooks:
+   * `data-list--collapse-toggle` - list toggle element hook to init ListCollapse click listener
+
+  Mandatory classes:
+   * `list--collapse` - placed on the outer container
+   * `list__item` - placed on the list items (eg li) in the list.
+
+  Optional Classes:
+   * `hide-inline-list--collapse-targets` - placed on the outer container to hide collapse the list by default. This class gets toggled when the toggle click listener fires
```

Comment:
I took a cue from [https://github.com/hmrc/assets-frontend/blob/master/assets/javascripts/modules/accordion.js](url)

I can make it less verbose if you like though